src/containers/BooksForm.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const BooksForm = () => {
+  const CATEGORIES = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  return (
+    <form id="Form">
+      <label htmlFor="Form">
+        Title
+        <input id="Title" type="text" />
+      </label>
+      <label htmlFor="Form">
+        Category
+        <select id="Category">
+          {CATEGORIES.map(category => (
+            <option key={parseInt(Math.random() * 1000, 10)} value={category}>{category}</option>
```

Comment:
**OPTIONAL**: Is the `key` going to be generated on every render? Remember that every time a component's `key` changes _[React will create a new component instance rather than update the current one](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recommendation-fully-uncontrolled-component-with-a-key:~:text=React%20will%20create%20a%20new%20component%20instance%20rather%20than%20update%20the%20current%20one.)._

Why not use the category as the key as it is going to be unique and stable?

Reply:
We applied your recommendation, thanks for this one! it's an awesome observation